sass/edx/_variables.scss

```diff
@@ -137,7 +138,7 @@ $body-color:    theme-color("dark") !default;
 // Style anchor elements.
 
 $link-color:            theme-color("primary") !default;
-$link-decoration:       none !default;
+$link-decoration:       underline !default;
```

Comment:
What was the thinking here, a11y concern?

Reply:
@HarryRein I'm going to revert this change as I'm not sure that it fits our current styling. I feel like links should be underlined in regular text, but that doesn't appear to be true on edX at the moment.